sdk/servicebus/service-bus/test/internal/batchReceiver.spec.ts

```diff
@@ -913,12 +913,7 @@ describe("Batching Receiver", () => {
 
         const result = await Promise.all([
           onDetachedCalledPromise,
-          delay(
-            Constants.defaultOperationTimeoutInMs * 1.5,
-            undefined,
-            undefined,
-            "ondetachednevercalled"
```

Comment:
looks like the test did not get translated fully?
`"ondetachednevercalled"` is missing in the update.

Reply:
Yeah if the abort controller is not passed (test was passing undefined), then the abort error is never going to be thrown, so it makes sense to remove it.

Reply:
![image](https://user-images.githubusercontent.com/10452642/188018435-f85ded85-6d65-499d-b823-56c686c85e37.png)
Looking deeper, it's not the error message, it's the 4th arg, `value`.

So, we should retain it for the proper expectation of the test.

Reply:
The `delay` function description looks odd because the `value` parameter has been deleted and it should not be of `string` type regardless. I'll look into it.